src/styles/header/_side.scss

```
@@ -0,0 +1,127 @@
+.side {
+  background-color: $croissant;
+  height: 840px;
+
+  &__container {
+    display: grid;
+    height: 87px;
+    grid-template-columns: 160px 110px 50px;
+    grid-template-areas: "tel button change" " number number number";
+    justify-content: end;
+    align-items: baseline;
+    padding: 60px 59px 0 0;
+    gap: 20px;
+    
+    text-align: end;
+  }
+
+  &__number-tel {
+    grid-area: tel;
+    text-decoration: none;
+    color: $chocolate;
+
+    transition: color 1s;
+
+    &:hover {
+      color: $cream;
+    }
+  }
+
+  &__number-button {
+    grid-area: button;
+    padding: 5px 15px 4px 18px;
+    width: 110px;
+    height: 30px;
+    
+    border: 2px solid $cream;
+    border-radius: 40px;
+    
+    background-color: $cream;
+    color: $chocolate;
+    
+    text-align: center;
+    
+    transition: background-color 1s;
+
+    cursor: pointer;
+
+    &:hover {
+      border: 2px solid $cream;
+      background-color: $milk;
+    }
+  }
+
+  &__change {
+    grid-area: change;
+  }
+
+  &__change-language-name {
+    border: none;
+    color: $chocolate;
+    background-color: $croissant;
+    
+    cursor: pointer;
+  }
+
+  &__number-container {
+    grid-area: number;
+  }
+
+  &__number {
+    position: relative;
+    padding-top: 130px;
+  }
+  
+  &__number::before {
```

Вклади в number елемент
```suggestion
  &::before {
```